components/src/core/Drawer/DrawerNavGroup.tsx

```diff
@@ -70,39 +118,67 @@ export type DrawerNavGroupProps = {
     chevron?: boolean;
     content?: ReactNode;
     divider?: boolean;
+    nestedDivider?: boolean;
     fontColor?: string;
     iconColor?: string;
     items: NavItem[];
     onSelect?: Function;
     open?: boolean;
+    ripple?: boolean;
     title?: string;
     titleColor?: string;
+    hidePadding?: boolean;
+    activeItem?: string;
 };
 
-function NavigationListItem(item: NavItem, props: DrawerNavGroupProps): ReactNode {
-    const { title, subtitle, icon, statusColor, onClick, active } = item;
-    const { divider = true } = props;
-    if (!title && !icon) {
-        return null;
-    }
+// renderer function for each nav item / nested nav item
+function NavigationListItem(
+    item: NavItem | NestedNavItem,
+    props: DrawerNavGroupProps,
+    depth = 0,
+    expanded = false
+): ReactNode {
+    const {
+        onClick,
+        title,
+        subtitle,
+        items: subItems,
+        divider: itemDivider,
+        onClickIcon,
+        itemID,
+        chevron: itemChevron,
+    } = item;
+    const icon = (item as NavItem).icon;
+    const { divider: groupDivider = true, nestedDivider } = props;
 
     const classes = useStyles(props);
     const theme = useTheme();
+    // @ts-ignore
+    const primary50Color = theme.palette.primary[50];
     const {
-        // @ts-ignore
-        activeBackgroundColor = theme.palette.type === 'light' ? theme.palette.primary[50] : theme.palette.primary.main,
+        activeBackgroundColor = theme.palette.type === 'light' ? primary50Color : theme.palette.primary.main,
         activeFontColor = theme.palette.type === 'light'
             ? theme.palette.primary.main
             : theme.palette.primary.contrastText,
         activeIconColor = theme.palette.type === 'light'
             ? theme.palette.primary.main
             : theme.palette.primary.contrastText,
-        fontColor,
-        chevron,
+        fontColor = theme.palette.text.secondary,
```

Comment:
why secondary and not primary?

Reply:
Update to the 4.0.0 of the theme and see if the updated font color is as expected

Reply:
> why secondary and not primary? 

Ideally we want gray[500] as default for inactive menu items. Right now primary gives black[500], and secondary gives black[300] -- so text.secondary is slightly "better". Or should I just do `Colors.gray[500]`?

Reply:
So why gray[500] over black [500]? Isn't black[500] our default text color? Just making sure this is something we want to treat as a special case. This might also be something we could update in the theme Overrides

Reply:
This will be a Kyle question... let me message him